firefox_ios/views/client_counts.view.lkml

```diff
@@ -0,0 +1,60 @@
+include: "//looker-hub/firefox_ios/views/baseline_clients_daily_table.view.lkml"
+
+view: client_counts {
+  extends: [baseline_clients_daily_table]
+
+  dimension_group: since_first_seen {
+    type: duration
+    description: "Amount of time that has passed since the client was first seen."
+    sql_start: ${TABLE}.first_seen_date ;;
+    sql_end: ${TABLE}.submission_date ;;
+    intervals: [
+      day, week, month, year
+    ]
+  }
+
+  dimension: have_completed_period {
+    type: yesno
+    description: "
+      Only for use with cohort analysis.
+      Filter on true to remove the tail of incomplete data from cohorts.
+      Indicates whether the cohort for this row have all had a chance to complete this interval.
+      For example, new clients from yesterday have not all had a chance to send a ping for today."
+    sql:
+      DATE_ADD(
+        {% if client_counts.first_seen_date._is_selected %}
+          DATE_ADD(DATE(${client_counts.first_seen_date}), INTERVAL 1 DAY)
+        {% elsif client_counts.first_seen_week._is_selected %}
+          DATE_ADD(DATE(${client_counts.first_seen_week}), INTERVAL 1 WEEK)
```

Comment:
When I pivot by first_seen_week, I see only one line plotted, corresponding to the earliest available week. I don't understand why I don't see other lines.

Reply:
Your example Look works: https://mozilla.cloud.looker.com/looks/9

It looks like the filters there are "in the past" whereas the example cohort analysis here is using "matches (advanced)".

Reply:
Okay this one was an error on my end - I changed the filters to use "ago", which I thought would be the same as "x completed weeks", but instead it's an exact week that is 8 weeks ago. Changing it to "in the past 8 weeks" shows the correct data. I'll update the filters.